src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 
 import BaseChannelItem from './BaseChannelItem';
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
-import {BaseChannelItemTypeProps} from '../../../types/component/AnonymousChat/BaseChannelItem.types';
+import useChannelHook from '../../hooks/screen/useChannelHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to use new hooks `useChannelHook` and `useUserAuthHook`. Ensure that these hooks are correctly implemented and tested, as they can significantly affect the functionality of the component.

```diff
- import useProfileHook from '../../hooks/core/profile/useProfileHook';
- import {BaseChannelItemTypeProps} from '../../../types/component/AnonymousChat/BaseChannelItem.types';
+ import useChannelHook from '../../hooks/screen/useChannelHook';
+ import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->